src/node_buffer.cc

```diff
@@ -19,7 +19,11 @@
   } while (0)
 
 #define ARGS_THIS(argT)                                                     \
-  Local<Object> obj = argT;                                                 \
+  Handle<Value> _obj = argT;                                                \
```

Comment:
Having both `_obj` and `obj` as `Local` causes test failures. It was my understanding that you could point any number of `Local`s at the same object, but I guess not?